TL improvement (#523)
07th-mod/higurashi-console-arcs: Update/omot_002_choice2.txt

	if (GetLocalFlag(LConsoleArc)==10) {
		ModPlayVoiceLS(4, 6, "ps2/06/210600101", 540, TRUE);
		OutputLine(NULL, "「………か、かか、カレーライス…、",
				NULL, "\"...C-c-curry and rice, umm...", Line_WaitForInput);
		ModPlayVoiceLS(4, 6, "ps2/06/210600102", 540, TRUE);
		OutputLine(NULL, "大盛、４枚……っ！」",

> 				NULL, " Make that four large helpings, please...!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " I'll take four large helpings, please...!\"", Line_Normal);
```

	} else {
		ModPlayVoiceLS(4, 6, "ps3/s14/06/vt4ba_shion1001", 256, TRUE);
		OutputLine(NULL, "「………マ、マスター、かか、カレーライス…、",

> 				NULL, "\"...E-e-excuse me sir, I-I'll have a curry and rice...", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"...E-e-excuse me ma'am, I-I'll have curry and rice...", Line_WaitForInput);
```

		ModPlayVoiceLS(4, 6, "ps3/s14/06/vt4ba_shion1002", 256, TRUE);
		OutputLine(NULL, "大盛で、４枚…っ！」",

> 				NULL, " And make that four large helpings, please...!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " And make it four large helpings, please...!\"", Line_Normal);
```

//r……とっさとはいえ、その意味不明な珍妙回答にクラス全員が『はい？？』とあっけに取られた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "……とっさとはいえ、その意味不明な珍妙回答にクラス全員が『はい？？』とあっけに取られた。",

> 			NULL, "...Of course her answer was a stopgap, but the whole class was dumbfounded by that unusual answer.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...She was just making something up on the spot of course, but the whole class was dumbfounded by that unusual answer.", Line_Normal);
```
